src/textual/strip.py

```diff
@@ -39,8 +54,13 @@ def __init__(
         self._cell_length = cell_length
         self._divide_cache: FIFOCache[tuple[int, ...], list[Strip]] = FIFOCache(4)
         self._crop_cache: FIFOCache[tuple[int, int], Strip] = FIFOCache(4)
+        self._style_cache: FIFOCache[Style, Strip] = FIFOCache(4)
```

Comment:
Is there something in the context that makes it clear what `4` is for?

Reply:
It's fairly arbitrary, predicated on observation that there are relatively few combinations of inputs.